src/main/scala/tech/cryptonomic/conseil/tezos/michelson/renderer/MichelsonRenderer.scala

```diff
@@ -52,7 +52,7 @@ object MichelsonRenderer {
       self
         .map(_.render())
         .mkString(" ;\n")
-        .lines
+        .linesIterator
```

Comment:
Really interesting history here, linesIterator was deprecated because it had the same functionality at lines, then undeprecated because lines conflicted with other Scala functions. How does this impact aggregation?

Reply:
it does not impact aggregation directly, but because of that travis did not want to build the project and run tests